test/AltoMare.test.js

```
@@ -0,0 +1,222 @@
+            try {
+                altoMare.validate("product", {});
+                assert(false, "Should have thrown error after unregistration");
+            } catch (error) {
+                assert(true, "Unregistered schema correctly rejected");
+            }
+        });
+
+        it('should retrieve premium user schema', () => {
+            const schema = altoMare.get("premiumUser");
+            assert(
+                schema && schema.email && schema.subscriptionTier,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)